tests/receivers/smartagent/prometheus-exporter/prometheus_exporter_test.go

```diff
@@ -18,13 +18,24 @@
 package tests
 
 import (
+	"path"
 	"testing"
 
 	"github.com/signalfx/splunk-otel-collector/tests/testutils"
 )
 
+var httpd = []testutils.Container{testutils.NewContainer().WithContext(
```

Comment:
nit, mind adding to the test directly since not shared?

Reply:
Sure thing.

Reply:
Fixed.